docs/gc.md

```diff
@@ -141,12 +141,14 @@ Object ages are tracked for each region with a maximum of 24 possible generation
 
 ![The diagram is explained in the surrounding text](./cr/balanced_regions.png "Region structure and characteristics found in the object heap")
 
- The *eden* space is a set of regions of age 0, which contain the newest objects allocated. Regions of intermediate age serve as survivor regions to receive live objects moved from younger regions. When the eden space is full, a partial garbage collection (PGC) runs to reclaim space, typically by using a *copy forward* operation. In specific cases, a *mark-compact* operation might be used, for example, when there are not enough free survivor regions available. The PGC is a stop-the-world operation, after which new regions are allocated to create a new eden space. A PGC always includes the eden space, but might include older regions. Objects within collectible regions of intermediate age N are evacuated to survivor regions of age N+1. Regions of age 24 are included in PGC collection sets in order to defragment them.
+ The *eden* space is a set of regions of age 0, which contain the newest objects allocated. Regions of intermediate age serve as survivor regions to receive live objects moved from younger regions. When the eden space is full, a partial garbage collection (PGC) runs to reclaim space, typically by using a *copy forward* operation. In specific cases, a *mark-compact* operation might be used, for example, when there are not enough free survivor regions available. The PGC is a *stop-the-world* (STW) operation, after which new regions are allocated to create a new eden space. A PGC always includes the eden space, but might include older regions. Objects within collectible regions of intermediate age N are evacuated to survivor regions of age N+1. Regions of age 24 are included in PGC collection sets in order to defragment them.
```

Comment:
Those couple of sentences about intermediate ages are not quite true. 
Firstly, survivor are both semi-populated regions (of ages from 1 to max age 24) and empty regions, if needed (and often are needed).
Secondly, objects from a region age N are evacuated into another region of same age N (or empty region) and then ages of all regions across heap (whether used as survivor or not) are incremented by 1. An exceptions is that the age of max age regions is never incremented (even though the may be part of PGC collection set).

Reply:
One more query @amicic. We mention that occasionally Balanced needs to run a global GC as an explicit GC. This is not in our table where we discuss heap configuration and GC cycles/ops. I think we ought to add it for completeness or are you happy that we discuss it only in the main Balanced section?

Reply:
This item already discussed. We won't put it in the table but we do mention it in the main section as a "rare event".